### DISCUSSION TITLE
Embedding videos

### DISCUSSION THREAD

**Discussion by xldrkp** [singlesourcepub/community#4] [General]
[![cartoon0](https://user-images.githubusercontent.com/1590241/119646050-ff82f180-be1e-11eb-94b6-05a60fc50d74.png)](https://www.youtube.com/watch?v=HUBNt18RFbo)

Does not work. But images work, wrapped with a link to the video.

**Comment by kthth** [comment]
Please let me join your group…

**Comment by xldrkp** [comment]
I leave this discussion here. It was a very early test how we could embed videos for the Show&Tell. As one can see, it would work with a linked still from a video.
